.changeset/thirty-ravens-study.md

```diff
@@ -0,0 +1,7 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+<!-- START components/app-header -->
+`AppHeader` - return `close` callback to the `:globalActions` and `:utilityActions` named blocks so the actions can be hidden programmatically.
```

Comment:
```suggestion
`AppHeader` - return `close` callback to the `:globalActions` and `:utilityActions` named blocks so the menu actions can be hidden programmatically.
```

Reply:
It may be helpful also clarify that this is relevant to the small/mobile view? (Just a thought. If you think it's not necessary, that's fine.)